plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -60,7 +60,7 @@ func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
 	return getAllTolerationPreferNoSchedule(tolerations), nil
 }
 
-// ComputeTaintTolerationPriority prepares the priority list for all the nodes based on the number of intolerable taints on the node
```

Comment:
LGTM, @davidopp Could you please add lgtm and approve label whenever you have time.